SocketIOClientSwift/SocketEngine.swift

```diff
@@ -350,8 +350,10 @@ public final class SocketEngine: NSObject, WebSocketDelegate {
     // We had packets waiting for send when we upgraded
     // Send them raw
     private func flushWaitingForPostToWebSocket() {
-        for msg in postWait {
-            ws?.writeString(msg)
+        if let ws = ws {
```

Comment:
Is the optimization really worth it? I'd prefer to keep as is, unless you've determined this is a bottleneck